.github/ISSUE_TEMPLATE/outdated-report.yml

```diff
@@ -0,0 +1,45 @@
+name: Package Outdated Report
+description: A package is outdated and no new version have been submitted for moderation.
+title: "(packageName) "
+labels: ["01 - _Triaging", "Outdated"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to report an outdated package. To validate the request please fill out the necessary items.
+
+        Make sure that the title of the issue starts with the package identifier that is outdated, inside paranthesis.
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Before continuing, make sure that you have done the following.
+      options:
+        - label: I have checked the moderation queue that no new version have been submitted (only visible when logged in), or the submitted package version have failed the Automated Checks.
+          required: true
+        - label: I have verified that the new version is a stable release.
+          required: true
+        - label: I have looked to see if there are any Bug reports that prevents a new version being submitted.
```

Comment:
Should this also ask if there are any preexisting outdated report issues?

Reply:
hmm, yeah that is a good point. I'll get that updated.